### PR TITLE
ci: Add concurrency

### PR DESCRIPTION
**Pull request by GeraltRiv**
PP-851

### CODE REVIEW

File: .github/workflows/bank-sdk.publish.docs.yml [hunk 1]
```diff
@@ -11,6 +11,10 @@ on:
         required: true
   workflow_dispatch:
 
+concurrency:
```
Comment (zladzeyka) [inline .github/workflows/bank-sdk.publish.docs.yml:14]: [Suggestion]: @GeraltRiv it's a part of release workflow, let's skip concurrency for `*.publish.docs.yml`  files

**Review by zladzeyka — CHANGES_REQUESTED** [review]
@GeraltRiv Thank you very much 🎉
I have one suggestion for excluding the concurrency from publishing docs below:

File: .github/workflows/bank-sdk.publish.docs.yml (hunk 1 above)
Comment (GeraltRiv) [reply .github/workflows/bank-sdk.publish.docs.yml:14]: Fixed

Comment (GeraltRiv) [reply .github/workflows/bank-sdk.publish.docs.yml]: Fixed

**Pull request merged**